conftest.py

```diff
@@ -0,0 +1,7 @@
+def pytest_addoption(parser):
```

Comment:
The Docker tests are ridiculously slow (~1 hour), take a bunch of disk space, and are flaky by nature. So let's not run them by default (we have mocks as well as the Docker integration tests).